core/iwasm/fast-jit/cg/x86-64/jit_codegen_x86_64.cpp

```diff
@@ -5176,20 +5866,41 @@ jit_codegen_gen_native(JitCompContext *cc)
                         GOTO_FAIL;
                     break;
 
+                case JIT_OP_I32CASTF32:
+                    LOAD_2ARGS();
+                    CAST_R_R(F32, I32, f32, i32, int32);
+                    break;
+
+                case JIT_OP_I64CASTF64:
+                    LOAD_2ARGS();
+                    CAST_R_R(F64, I64, f64, i64, int64);
+                    break;
+
+                case JIT_OP_F32CASTI32:
+                    LOAD_2ARGS();
+                    CAST_R_R(I32, F32, i32, f32, float);
+                    break;
+
+                case JIT_OP_F64CASTI64:
+                    LOAD_2ARGS();
+                    CAST_R_R(I64, F64, i64, f64, double);
+                    break;
+
                 default:
                     jit_set_last_error_v(cc, "unsupported JIT opcode 0x%2x",
                                          insn->opcode);
                     GOTO_FAIL;
             }
 
             if (err_handler.err) {
-                jit_set_last_error_v(
-                    cc, "failed to generate native code for JIT opcode 0x%02x",
-                    insn->opcode);
+                jit_set_last_error_v(cc,
+                                     "failed to generate native code for JIT "
+                                     "opcode 0x%02x, ErrorCode is %u",
+                                     insn->opcode, err_handler.err);
                 GOTO_FAIL;
             }
 
-#if CODEGEN_DUMP != 0
+#if WASM_ENABLE_FAST_JIT_DUMP != 0
```

Comment:
Same as above.

Reply:
there will be a compiling error around `dump_native`

Reply:
If the two places of CODEGEN_DUMP were not changed (L4824 and here), there should be no compilation error?
When the two places are changed and some other places of CODEGEN_DUMP are not changed, there will be some extra machine code dumped, it is a little strange.

Reply:
It is a quick fix and maybe not very accurate and reasonable but workable. 

The compiling error is able to be reproduced with the code of branch `dev/fast_jit` and `-DWAMR_BUILD_FAST_JIT_DUMP=0`